module-4/backend/week_one.md

```diff
@@ -1,21 +1,71 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+>That I can make the connection between Ruby and JS and apply those changes pretty well.
+
 2. What are some tools to help debug JavaScript code?
+
+>Debugger, console.log(), breakpoints
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+>Mocha, Chai, npm, node modules
+
 4. What is the syntax for invoking a function?
+
+>functionName()
+
 5. What is CORS?
+
+>Cross Origin Resource Sharing, allows resources to be shared to another domain,
+outside of the current domain.
+
 6. How do we enable CORS?
+
+>As part of the headers in the ajax call => Access-Control-Allow-Origin
+
 7. What's `this` in JavaScript?
+
+>Similar to ruby self. it refers to the current spot in the code that it is referring to in the scope fo the function.
+
 8. What is Webpack and why is it useful?
+
+>A module bundler. Its main purpose is to bundle JavaScript files for usage in a browser, yet it is also capable of transforming, bundling, or packaging just about any resource or asset.
+
 9. When/why do you want to use event delegation?
+
+>When you may have multiple 'buttons' that require an event listener. This way
```

Comment:
Also when you may want to listen to an event that doesn't yet exist on the page. If you place an event on the parent element, event bubbling will cause the parent to catch events that happen on any children that are currently there or will be there in the future.